app/javascript/utils/form/validations.js

```diff
@@ -136,4 +138,15 @@ export const validateLeaseCurrency = (value) => {
   )
 }
 
+// format currency in one call instead of in different mappers
+export const convertCurrency = (values) => {
```

Comment:
please add some unit tests for this function. We should have some on the old currency conversion function that you can repurpose.

Reply:
Added a test around this to illustrate